libs/ui/src/display_settings_manager_context.tsx

```diff
@@ -13,12 +13,16 @@ export interface ThemeManagerContextInterface {
 
   /** Switches over to the specified size theme. */
   setSizeMode: (mode: SizeMode) => void;
+
+  /** Toggles a full screen overlay that visually blocks underlying elements but keeps them in the accessibility tree */
+  setIsVisualModeDisabled: (isDisabled: boolean) => void;
```

Comment:
This change is related to a later commit https://github.com/votingworks/vxsuite/pull/4518/commits/d312ab52cdd5aaaecba67fda750586b04a71ec3d but accidentally committed here